codegen/src/main/java/software/amazon/awssdk/codegen/internal/DocumentationUtils.java

```diff
@@ -36,6 +36,8 @@
 
     private static final String DEFAULT_GETTER_PARAM = "The value of the %s property for this object.";
 
+    private static final String DEFAULT_EXISTENCE_CHECK = "Returns true if the %s property has a meaningful value.";
```

Comment:
Hah, this might be a little too honest. How about something like "Returns true if the %s property was specified by the sender (it may be empty), or false if the sender did not specify the value (it will be empty). For responses returned by the SDK, the sender is the AWS service."

Reply:
Perfect, will change.

Reply:
Thanks! We might also add a note to the getter that the caller can check the "has" method to see whether a value was sent in this field.

Reply:
Both documentation changes made in 48b3879